agimus_controller_ros/agimus_controller_ros/trajectory_weights_parameters.yaml

```
@@ -13,6 +13,10 @@ trajectory_weights_params:
             "generic_trajectory",
           ],
         ]
+  mask:
+    type: bool_array
+    default_value: [True, True, True, True, True, True]
+    description: "Mask for the space sine wave DoF to control."
```

In all cases we will still have a cost on the rotation of the end-effector pose so I'll suggest something like 
```suggestion
    description: "Mask for the cartesian space sine wave DoF to update from the Inverse Kinematic."
```